### PR TITLE
Add xilemsvg example

### PR DESCRIPTION
**Pull request by raphlinus**
This is a proof of concept that uses the common xilem_core to generate vector graphics content rendered as SVG DOM nodes in the browser.

This could plausibly be made into an actual library, and it's likely that an evolution of this will be used to create interactive illustrations, for example in the raphlinus blog.

### CODE REVIEW

File: crates/xilemsvg/dist/index.html
Comment (DJMcNab) [inline crates/xilemsvg/dist/index.html:1]: This dist folder should be `gitignore`d, I think

Comment (nicoburns) [reply crates/xilemsvg/dist/index.html:1]: And basic instructions for building/running should be included in it's place (FYI, with the current code you must run a web server in the dist directory for the links to work properly).

**Comment by raphlinus** [comment]
Marking this as ready for review even though it contains a number of rough edges, to maintain forward momentum.

**Comment by derekdreery** [comment]
Ahh sorry I merged the xilem_html PR first. You'll be listed as a co-author. :P

**Comment by xorgy** [comment]
‘Let's just close it’ — @raphlinus

**Pull request closed without merging**